toolbox

```diff
@@ -1085,6 +1085,12 @@ EOF
         fi
     fi
 
+    echo "$base_toolbox_command: removing blocker file" >&3
+
+    cont_toolbox_runtime_dir="/run/host/run/user/1000/toolbox"
+
+    rm --force "$cont_toolbox_runtime_dir"/toolbox_init_lock 2>&3
```

Comment:
Umm... since `XDG_RUNTIME_DIR` is shared between the host and all containers, this means that different `podman start` invocations by different instances of `toolbox enter` can step on each others toes by deleting the same file. That would be bad. :)

I think it's a practical solution to use the same file for all containers. Otherwise, if it was an unique path for each container, then we'd have to somehow pass the path to the entry-point during `podman create` and then dig it back up in `run()`. That's a bit painful to implement.

However, as long as we have a global path for locking, we have to somehow prevent different instances from colliding against each other. Maybe we do need `flock` here after all.

Reply:
Backwords compatibility is another hairy issue. Older Toolbox containers didn't have `toolbox init-container` as their entry-point. They won't have anything to cancel the waiting, and cause `toolbox enter` to get stuck. We need to handle this case.